### ISSUE TITLE
  I have some doubts about "if rot90:    img = img.transpose(1, 0, 2)".

### ISSUE THREAD

**Issue by Tangjian2344** [thstkdgus35/EDSR-PyTorch#275]
Dear Author
    I think the effect of this code is to fold the image diagonally instead of rotating it 90 degrees。

**Comment by Matthewhouch** [comment]
Yeah, you're right, it's a flip over diagonal, but I think it can be seen as a kind of 90 degrees rotation.
